server/plugins/client/bundle.js

```diff
@@ -31,6 +31,14 @@ function bundleClient (hoodieClientPath, bundleTargetPath, config, callback) {
   var plugins = [
     path.resolve('hoodie/client')
   ].filter(checkModule)
+
+  var thirdPartyPlugins = require('hoodie.plugins')
```

Comment:
`require('hoodie.plugins')` looks for a module with the name `hoodie.plugins` that would need to be listed in hoodie’s [package.json dev/dependencies](https://github.com/hoodiehq/hoodie/blob/b2f16b827148ec8478571f42572292d635abc847/package.json#L13-L49).

Instead you want to look into `config.plugins`. We don’t currently have a `plugins` option, so you would need to add that. Maybe not for the CLI usage, but for the [hapi usage](https://github.com/hoodiehq/hoodie#hapi-plugin). The default value should be `[]` and you can set it in https://github.com/hoodiehq/hoodie/blob/master/cli/hoodie-defaults.js

To test all this, I suggest you create a new Hoodie app as described in http://docs.hood.ie/en/latest/guides/quickstart.html, and then link your local `hoodie` repository into your app, with `npm link /path/to/hoodie`. That way you can restart your app and the changes you do to your local `hoodie` repository will directly apply.

In your app’s `package.json`, add `"hoodie": {"plugins": ["hoodie-example-plugin"]}`, and install `hoodie-example-plugin` as a dependency.

Now if you restart your app, `config.plugins` should be `['hoodie-example-plugin']`, so you can do `var thirdPartyPlugins = config.plugins

I know it’s quite a lot, I hope it’s not too overwhelming?